src/Moneybird.Net/Models/SalesInvoices/SalesInvoiceSendOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Moneybird.Net.Misc;
+using System.Text.Json.Serialization;
+
+namespace Moneybird.Net.Models.SalesInvoices
+{
+    public class SalesInvoiceSendOptions
+    {
+        [JsonPropertyName("delivery_method")]
+        public DeliveryMethod DeliveryMethod { get; set; }
+
+        [JsonPropertyName("sending_scheduled")]
+        public bool SendingScheduled { get; set; }
+
+        [JsonPropertyName("sending_ubl")]
+        public bool SendingUbl { get; set; }
```

Comment:
The documentation shows that this should be 'deliver_ubl' instead of 'sending_ubl'. Or did you find that the documentation is wrong in this regard?

Reply:
Correct! This was a discrepancy on my part, thanks for pointing out.